src/kubeseal_auto/secrets/creation.py

```
@@ -0,0 +1,143 @@
+    """
+    key_file = Path("tls.key")
+    cert_file = Path("tls.crt")
+
+    missing_files = []
+    if not key_file.exists():
+        missing_files.append("tls.key")
+    if not cert_file.exists():
+        missing_files.append("tls.crt")
+
+    if missing_files:
+        raise click.ClickException(f"Required TLS file(s) not found in current directory: {', '.join(missing_files)}")
```

Ruff (0.14.7)

Avoid specifying long messages outside the exception class

(TRY003)